src/network/networkd-link.c

```diff
@@ -3376,6 +3376,9 @@ int link_save(Link *link) {
                 char **dhcp_domains = NULL;
                 unsigned j;
 
+                fprintf(f, "REQUIRED_FOR_ONLINE=%s\n",
+                        true_false(link->network->required_for_online));
```

Comment:
hmm, not that it matters, but when serializing state like this we never used "true_false()" so far, but either "yes_no()" or "one_zero()". Given that parse_boolean() actually can parse all three it doesn't really matter, but just to keep this in sync with the rest, please use yes_no()